### PR TITLE
Fix path to grub.cfg file

### PR DESCRIPTION
**Pull request by jimmykarily**

### CODE REVIEW

File: content/en/docs/Installation/automated.md [hunk 1]
```diff
@@ -128,7 +128,8 @@ total 778M
 $ IMAGE=<scheme://host[:port]/path[:tag]>
 $ mkdir -p files-iso/boot/grub2
 # You can replace this step with your own grub config. This GRUB configuration is the boot menu of the ISO
-$ wget https://raw.githubusercontent.com/kairos-io/kairos/master/overlay/files-iso/boot/grub2/grub.cfg -O files-iso/boot/grub2/grub.cfg
+$ wget https://raw.githubusercontent.com/kairos-io/packages/main/packages/static/grub-config/files/grub.cfg -O files-iso/boot/grub2/grub.cfg
```
Comment (Itxaka) [inline content/en/docs/Installation/automated.md:131]: Should we use the grub from the livecd package?

Comment (jimmykarily) [reply content/en/docs/Installation/automated.md:131]: which one?

Comment (Itxaka) [reply content/en/docs/Installation/automated.md:131]: https://raw.githubusercontent.com/kairos-io/packages/main/packages/livecd/grub2/config/grub_live_bios.cfg

Comment (jimmykarily) [inline content/en/docs/Installation/automated.md:131]:
```suggestion
$ wget https://raw.githubusercontent.com/kairos-io/packages/main/packages/livecd/grub2/config/grub_live_bios.cfg -O files-iso/boot/grub2/grub.cfg
```

Comment (jimmykarily) [reply content/en/docs/Installation/automated.md]: done

**Review by Itxaka — APPROVED** [review]

**Pull request merged**